scss/_utilities.scss

```diff
@@ -542,6 +542,7 @@ $utilities: map-merge(
           "black-50": rgba($black, .5), // deprecated
           "white-50": rgba($white, .5), // deprecated
           "reset": inherit,
+          "primary": var(--#{$prefix}link-hover-color),
```

Comment:
Isn't it better to use directly `$accessible-orange`?

Reply:
Not sure about that since we want it to be `$brand-orange` on dark backgrounds, right ?

Reply:
Ah yes I was too focused on the weird usage of "link-hover-color". So OK to use a CSS var to allow the switch of orange color but maybe we could create a specific CSS var rather than using the one for the hovered links.

Reply:
I'm not sure about the name I gave, it could be discussed as well